test/cql-pytest/test_in_operator.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+import random
+import string
+from typing import List, Any
+
+import pytest
+from cassandra import InvalidRequest
+from cassandra.cluster import Session, NoHostAvailable
+
+from util import unique_name
+
+CARTESIAN_PRODUCT_LIMIT = 100
```

Comment:
I think it is bad style to put a constant only used once in the top of the source file, where nobody knows what it relates to. Please move it in front of the two tests that actually uses it, so that readers of that test immediately see the constant as well.
I'd also add a comment where this constant comes from. See https://github.com/scylladb/scylla/issues/8882 for an explanation.

Reply:
Done